templates/event.php

```diff
@@ -84,5 +72,26 @@
 			</tbody>
 		</table>
 	</div>
+	<details class="event-stats-summary">
```

Comment:
Just a detail, but I noticed the element moves on hover:

https://github.com/WordPress/wporg-gp-translation-events/assets/550401/f65ac810-cde2-4c70-9f2b-a569d062158a